install-raspbian.sh

```diff
@@ -114,6 +115,29 @@ else
     tar -xzvf /tmp/photobooth-latest.tar.gz -C /var/www/html/
 fi
 
+if [ -f "/etc/nginx/sites-enabled/default" ]; then
+    info "### Enable PHP in NGINX"
+    cp /etc/nginx/sites-enabled/default ~/nginx-default.bak
+    sed -i 's/^\(\s*\)index index\.html\(.*\)/\1index index\.php index\.html\2/g' /etc/nginx/sites-available/default
+    sed -i '/location ~ \\.php$ {/s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
+    sed -i '/include snippets\/fastcgi-php.conf/s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
+    sed -i '/fastcgi_pass unix:\/run\/php\//s/^\(\s*\)#/\1/g' /etc/nginx/sites-available/default
+    sed -i '/.*fastcgi_pass unix:\/run\/php\//,// { /}/s/^\(\s*\)#/\1/g; }' /etc/nginx/sites-available/default
+
+    info "### Testing NGINX config"
+    /usr/sbin/nginx -t -c /etc/nginx/nginx.conf &>/dev/null && RESULT='config test ok' || RESULT='config test failed'
```

Comment:
This should be changed to:
```bash
/usr/sbin/nginx -t -c /etc/nginx/nginx.conf
systemctl reload nginx
```
... as the test run will output useful information what is wrong with the config.